ext/ffi/prelude.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
+
+/* Exact integral types.  */
+
+/* Signed.  */
+typedef signed char int8_t;
```

Comment:
Hmm, since this prelude is already deciding that `char` is 8 bits, `short int` is 16 bits etc. and there's no 32 bit support, should these just be inlined back into the `jit_trampoline.rs` file?

There are probably systems where these typedefs are invalid (I presume the `stdint.h`'s actual contents is platform dependent to handle that), but Deno probably does not have support for that anyway.

Reply:
> Deno probably does not have support for that anyway.

Added const assertions for the assumptions made.

It'd be nice to just have an extendable prelude, updating test assertions is a pain. besides, we'll need it for Fast TypedArray params.

Reply:
Looks very good to me <3 Explicitness is very good, and if any downstream errors present at least it'll be really explicit.